model/inviteModel.js

```diff
@@ -3,16 +3,17 @@ const Mongoose = require('mongoose');
 const inviteSchema = Mongoose.Schema({
     userID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'User ID must be specified']
     },
     eventID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Event ID must be specified']
     },
     responseType: {
         type: Number,
-        required: true,
-        default: 0
+        required: [true, 'Invite response must be specified'],
+        default: 0,
+        enum: [[0, 1, 2], 'Invalid response type value']
```

Comment:
Use already defined `ResponseTypeEnum` values here.

Reply:
How would i go about doing that? Like:
enum: [ [ResponseTypeEnum.Pending...], 'Error message...']
Like this?

Reply:
`ResponseTypeEnum['Pending']`, etc.